prqlc/prql-compiler/tests/integration/error_messages.rs

```diff
@@ -101,6 +101,24 @@ fn test_errors() {
     "###);
 }
 
+#[test]
+fn array_instead_of_tuple() {
+    // Particularly given this used to be our syntax, this could be clearer
+    // (though we do say so in the message, which is friendly!)
```

Comment:
```suggestion
```

Reply:
of course [786d90e](https://github.com/PRQL/prql/pull/3884/commits/786d90e262ab5795d5516267144f4d54d7840810)

Reply:
Thanks! (FYI you can also hit "Commit suggestion" in GH and it'll add a commit with the change for you :) )